### PR TITLE
Upgrade/enso6 dr

### PR DESCRIPTION
**Pull request by AbdullahiAbdulkabir**

### CODE REVIEW

File: app/Dynamics/Company/Comments.php [hunk 1]
```diff
@@ -4,17 +4,23 @@
 
 use Closure;
 use LaravelEnso\Comments\Models\Comment;
+use LaravelEnso\Companies\Models\Company;
 use LaravelEnso\DynamicMethods\Contracts\Method;
 
 class Comments implements Method
```
Comment (GITmanuela) [inline app/Dynamics/Company/Comments.php:10]: Relation

File: app/Dynamics/Company/Discussions.php [hunk 2]
```diff
@@ -3,18 +3,24 @@
 namespace App\DynamicRelations\Company;
 
 use Closure;
+use LaravelEnso\Companies\Models\Company;
 use LaravelEnso\Discussions\Models\Discussion;
 use LaravelEnso\DynamicMethods\Contracts\Method;
 
 class Discussions implements Method
```
Comment (GITmanuela) [inline app/Dynamics/Company/Discussions.php:10]: Relation

File: app/Dynamics/Company/Documents.php [hunk 3]
```diff
@@ -3,18 +3,24 @@
 namespace App\DynamicRelations\Company;
 
 use Closure;
+use LaravelEnso\Companies\Models\Company;
 use LaravelEnso\Documents\Models\Document;
 use LaravelEnso\DynamicMethods\Contracts\Method;
 
 class Documents implements Method
```
Comment (GITmanuela) [inline app/Dynamics/Company/Documents.php:10]: Relation

**Comment by oracast** [comment]
can we merge it?

**Comment by vmcvlad** [comment]
> can we merge it?

very soon

**Pull request merged**